payments/views/v0/urls.py

```diff
@@ -37,4 +38,9 @@
         name="checkout-callback",
     ),
     re_path("^", include(router.urls)),
+    path(
+        "checkout/result/",
```

Comment:
Moving this here changes the namespace of where this is in the URLconf. This is fine but as is it's causing a 500 error when trying to check out, because the `generate_checkout_payload` API tries to build the redirect URL without the namespace. (This is a lot of words to say "add a `v0:` to the start of the url in the generate_checkout_payload API".)